CHANGELOG.md

```diff
@@ -12,6 +12,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Add Renovate to track new tags.
 - Update `kube-vip` helm chart to `0.6.2`.
 - Update `kube-vip` to `0.8.4`.
+- Update `kube-vip` to `0.8.9`.
```

Comment:
The `kube-vip` PR hasn't been merged, yet.

Reply:
good shout, i was mixing my repos up :facepalm: i've removed that line